### ISSUE TITLE
Combobox: when using VoiceOver, down arrow gets announces (iOS)

### ISSUE THREAD

**Issue by wkeese** [ibm-js/deliteful#615]
On iOS when you turn on VoiceOver, and then click the Combobox's down arrow, it actually "focuses" (or something similar), and iOS announces "black down arrow".
